spark/src/main/scala/io/ddf/spark/content/RDDRow2ArrayString.scala

```diff
@@ -0,0 +1,21 @@
+package io.ddf.spark.content
+
+import io.ddf.DDF
+import io.ddf.content.{ConvertFunction, Representation}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.Row
+
+class RDDRow2ArrayString (@transient ddf: DDF) extends ConvertFunction(ddf) {
+
+  override def apply(representation: Representation): Representation = {
+    representation.getValue match {
+      case rdd: RDD[Row] =>
+        val rddArrString = rdd.map {
+          row => {
+            row.getString(0).split("[ ,]")
+          }
+        }.filter(row => row != null)
```

Comment:
if the row is null, there'll be NPE at line 15, so properly won't need to filter null at this line